PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -81,6 +81,9 @@ CorrectionSigma=`opts_GetOpt1 "--mcsigma" $@`
 RegName=`opts_GetOpt1 "--regname" $@`
 InflateExtraScale=`opts_GetOpt1 "--inflatescale" $@`
 
+MPPMode=`opts_GetOpt1 "--mpp-mode" $@`
+MPPMode=`opts_DefaultOpt $MPPMode "HCPStyleData"`
```

Comment:
Remove blank line preceding setting of MPPMode, and move the setting of its default value down several lines to where the defaults on other parameters are set.

Reply:
Ok.